content/exchange/artifacts/PrintNightmare.yaml

```diff
@@ -19,6 +19,7 @@ description: |
   - Time bounds to select files with a timestamp within time ranges
   - FileSize bounds
 
+  ![Sample output](PrintNightmare.png)
```

Comment:
I initially had it like that but the problem is that when one copies the artifact to Velociraptor it results in a broken image. So I found the best way is to copy the url from github

Reply:
I think my last commit will resolve the issue?